content/en/serverless/installation/nodejs.md

```diff
@@ -0,0 +1,204 @@
+  ```
+  Transform:
+	  DD::Serverless
+  ```
+2. In your `template.yml`, also add the following section:
+  ```   
+  Mappings:
+	  Custom:
+		  Datadog:
+        enableDDTracing: true
+			  flushMetricsToLogs: true
+			  # The Datadog Forwarder ARN goes here.
+			  forwarder:
+  ```
+  For more information on the Forwarder ARN, or to install the forwarder see the [official CloudFormation documentation][1].
+4. Redeploy your serverless application.
+
+
+[1]: https://console.aws.amazon.com/cloudformation/home#/stacks?filteringText=forwarder
+{{% /tab %}} --->
+{{% tab "AWS Console" %}}
+
+### Install the Datadog Lambda Library
+
+The Datadog Lambda Library can be imported as a layer. It’s ARN includes a region, language runtime, and version. Construct yours in the following format:
+
+```
+arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:<VERSION>
+```
+
+For example:
+
+```
+arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Python37:11
+```
+
+The Python Datadog Lambda Library supports Node8-10, Node10-x, Node12-x. For more information, see the [latest release][1].
+
+To install the Datadog Lambda Library:
+
+1. Navigate to the Lambda function to which you want to add the layer in your AWS console.
+2. Click on **Layers** on the main page of your function.
+3. Scroll down, and click on **Add a Layer**.
+4. Select the option to *Provide a layer version ARN*.
+5. Enter the Datadog Lambda Library ARN from the table above.
+6. Navigate to the Environment Variables section of your function and add a new `“DD_FLUSH_TO_LOG”` variable set to `“true”`.
+
+These steps need to be repeated for every function you wish to trace.
+
+### Instrument your Code
+
+Instrument your functions to ingest traces into Datadog:
+
+```
+const { datadog } = require('datadog-lambda-js');
+const tracer = require('dd-trace').init(); // Any manual tracer config goes here.
+
+// This function will be wrapped in a span
+const longCalculation = tracer.wrap('calculation-long-number', () => {
+    // An expensive calculation goes here
+});
+
+// This function will also be wrapped in a span
+module.exports.hello = datadog((event, context, callback) => {
+    longCalculation();
+
+    callback(null, {
+        statusCode: 200,
+        body: 'Hello from serverless!'
+    });
+});
+```
+
+### Subscribe the Forwarder to Log Groups
+
+You need the Datadog Forwarder to subscribe to each of your function’s log groups to send traces and enhanced metrics to Datadog.
+
+For more information on the Forwarder ARN, or to install the forwarder see the [documentation][1]. Make sure the Datadog Forwarder is in the same AWS region as the Lambda functions you are monitoring.
+
+1. To start, navigate to your AWS Dashboard for the Datadog Forwarder. Then, manually add a function trigger.
+2. Configure the trigger to be linked to your function’s CloudWatch Log Group, add a filter name (but feel free to leave the filter empty) and add the trigger:
```

Comment:
```suggestion
2. Configure the trigger to be linked to your function’s CloudWatch Log Group, add a filter name (but feel free to leave the filter empty) and add the trigger:
```

the colon at the end of this sentence makes me feel like there's supposed to be something after it